templates/irc/index.html

```diff
@@ -60,7 +60,7 @@ <h1 class="title-shadow-box">{{ self.title()}}</h1>
 					</div>
 					<div class="container">
 
-          <object width="900" height="700" data="http://webchat.mozilla-tunisia.org:9090/?channels=#mozillatunisia">
+          <object id="irc_frame" height="700" width="100%" data="http://webchat.mozilla-tunisia.org:9090/?channels=#mozillatunisia">
```

Comment:
since you added a new `id` attribute, then the style should be in a new css selector.